Gemfile

```diff
@@ -12,8 +12,10 @@ end
 gem 'rails', '~> 5.0.1'
 gem 'rails-controller-testing'
 
-gem 'sprockets', '~> 3.6.3'
-gem 'sprockets-rails', '~> 3.1.1'
+# Compile Sprockets assets concurrently in `assets:precompile`.
+# Ref: https://github.com/rails/sprockets/pull/469
+gem 'sprockets', github: 'wjordan/sprockets', ref: 'concurrent_asset_bundle_3.x'
```

Comment:
ahh, i see we have an open PR to get this into sprockets master, at which point we could presumably go back to not using a custom fork?